src/Tag/Base/ContentTag.php

```diff
@@ -4,61 +4,66 @@
 
 namespace Yiisoft\Html\Tag\Base;
 
+use Stringable;
 use Yiisoft\Html\Html;
+use Yiisoft\Html\NoEncodeStringableInterface;
 
 abstract class ContentTag extends NormalTag
 {
-    private bool $encode = true;
-    private bool $encodeSpaces = false;
+    private ?bool $encode = null;
     private bool $doubleEncode = true;
-    private string $content = '';
 
     /**
-     * @param bool $encode Whether to encode tag content. Defaults to `true`.
+     * @var string[]|Stringable[]
+     */
+    private array $content = [];
+
+    /**
+     * @param bool|null $encode Whether to encode tag content. Defaults to `null`.
```

Comment:
Need description for true/false/null.

Reply:
I think it should be only `true/false` mode. Without `null`.

Reply:
`null` currently means "automatic based on interface". Which value, false or true should work like that?

Reply:
Done.